lib/fmrest/spyke/relation.rb

```diff
@@ -5,6 +5,8 @@ module Spyke
     class Relation < ::Spyke::Relation
       SORT_PARAM_MATCHER = /(.*?)(!|__desc(?:end)?)?\Z/.freeze
 
+      ImpossibleValue = Object.new.freeze
```

Comment:
Curious why this is PascalCase rather than SCREAMING_SNAKE_CASE? (Not saying it's wrong, just curious)

Reply:
Lol, good question. I think I was thinking of other special values in Ruby like *Infinity* or *NaN*, which I misremembered as using PascalCase, probably because that's how they advertise themselves with `.inspect`, even though the actual constants don't:

```
irb(main):007:0> Float::INFINITY
=> Infinity
irb(main):008:0> Float::NAN
=> NaN
```

I'll change it.